src/slider/slider.component.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { Component } from "@angular/core";
+import { TestBed } from "@angular/core/testing";
+import { By	 } from "@angular/platform-browser";
```

Comment:
Maybe it's just github, but I'm seeing some weird spacing here
```suggestion
import { By } from "@angular/platform-browser";
```